content/en/blog/_posts/2019-03-12-kubeedge-k8s-based-edge-intro.md

```diff
@@ -0,0 +1,114 @@
+
+
+**<p align="center">You heard it right!</p>**
+**<p align="center">KubeEdge Cloud Core modules are open sourced!</p>**
+
+
+By open sourcing both edge and cloud modules KubeEdge brings a complete cloud vendor agnostic light weight heterogeneous edge computing platform. It is now ready to support building a complete kubernetes ecosystem for edge computing, exploiting most of the existing cloud native projects or software modules. This can enable a mini-cloud at edge to support demanding use cases like data analytics, video analytics, machine learning and more.
+
+## KubeEdge Architecture: Building Kuberenetes Native Edge computing!
+
+The core architecture tenet for KubeEdge is to build interfaces those are consistent with Kubernetes, be it in the cloud side or edge side. 
+
+<p align="center">
+<img src="../../../../static/images/blog/2019-03-12-kubeedge-k8s-based-edge-intro/kubeedge-highlevel-arch.png">
+</p>
+
+**Edged**: Manages containerized Applications at the Edge
+
+**EdgeHub**: Communication interface module at Edge. It is a web socket client responsible for interacting with Cloud Service for the edge computing. 
+
+**CloudHub**: Communication interface module at Cloud. A web socket server responsible for watching changes at the cloud side, caching and sending messages to EdgeHub.
+
+**EdgeController**: Manages the edge nodes. It is an extended kubernetes controller which manages edge nodes and pods metadata so that the data can be targeted to a specific edge node.
+
+**EventBus**: Handles the internal edge communications using MQTT. It is an MQTT client to interact with MQTT servers (mosquitto), offering publish and subscribe capabilities to other components.
+
+**DeviceTwin**: It is software mirror for devices that handles the device metadata. This module helps in handling device status and syncing the same to cloud. It also provides query interfaces for applications, as it interfaces to a lightweight database (SQLite).
+
+**MetaManager**: It manages the metadata at edge node. This is the message processor between edged and edgehub. It is also responsible for storing/retrieving metadata to/from a lightweight database (SQLite).
+
+Even if want to add more control plane module based on the architecture refinement and improvement (for example enhanced security), it is simple as it uses consistent registration and modular communication within these modules.
+
+
+**<p align="center">KubeEdge provides scalable light weight Kubernetes Native Edge Computing Platform which can work in offline mode.</p>** 
+
+**<p align="center">It helps simplified edge application development and deployment.</p>**
+
+**<p align="center">Cloud vendor agnostic and can run the cloud core modules on any compute node.</p>**
+
```

Comment:
the same

Reply:
Highlighting in between the blog these texts to call out that point. Aligned to center.